app/assets/stylesheets/newflow_layout.scss

```diff
@@ -1,13 +1,39 @@
-@import "font-awesome";
 @import "pattern-library";
+@import "font-awesome";
+@import "newflow_colors";
 
 html {
-    // font-family: 'HelveticaNeue-Bold', 'Helvetica Neue Bold', 'Helvetica Neue';
-    font-family: 'ArialMT', 'Arial';
+    font-family: 'Helvetica Neue';
 }
 
 body {
+    @include width-up-to($phone-max) {
+        .content {
+            padding-left: 0;
+            padding-right: 0;
+        }
+    }
+
     background-color: #fff;
+    color: var(--greyish-brown);
+}
+
+// utility classes
+.hidden {
+    display: none !important;
+}
+
+.main-menu {
+    box-shadow: none;
+    height: 2rem;
+    margin-top: 0.8rem;
+
```

Comment:
Extra line?
Is there an scss-lint step in the build?

Reply:
There is not. Removed it.